File: 0217-contains-duplicate/solution.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        numSet = set()
        
        for num in nums:
            if num not in numSet:
                numSet.add(num)
            else:
                return True
            
        return False

File: 0217-contains-duplicate/solution.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) 
    {
        unordered_map<int, int> numCount;
        for(int i = 0; i < nums.size(); i++)
        {
            numCount[nums[i]]++;
        }
        return numCount.size() != nums.size();
        
    }
};
